pkg/offline/offline.go

```diff
@@ -32,7 +26,10 @@ const (
 	maxRequeueAttempts = 3
 	// sendLimit is the maximum number of heartbeats, which will be sent at once
 	// to the WakaTime API.
-	sendLimit = 10
+	sendLimit = 24
```

Comment:
How is this different from `sendLimit` in `cmd/heartbeat/heartbeat.go`?

Reply:
Nothing, them have the same purpose. Just forgot to merge in a single one.